python/pyspark/sql/tests/pipelines/test_block_connect_access.py

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import unittest
+
+from pyspark.errors import PySparkException
+from pyspark.testing.connectutils import ReusedConnectTestCase
+from pyspark.sql.pipelines.block_connect_access import block_spark_connect_execution_and_analysis
+
+
+class BlockSparkConnectAccessTests(ReusedConnectTestCase):
```

Comment:
new tests should be registered in `dev/sparktestsupport/modules.py`, otherwise they are skipped

Reply:
Do we plan to also test this file in classic mode?

Reply:
This should only be tested in connect mode – do I need to add something to the file to set that up?